src/element.ts

```diff
@@ -260,6 +260,13 @@ export abstract class Element {
     return this.attrs[name];
   }
 
+  /** Return associated SVGElement. */
+  getSVGElement(suffix: string = ''): SVGElement | undefined {
+    const id = prefix(this.attrs.id + suffix);
+    const element = document.getElementById(id);
+    if (element) return element as unknown as SVGElement;
+  }
+
```

Comment:
@0xfe are you fine with this approach rather than saving the `SVGElement` in `attrs.el`?
If yes I will merge once the review from aaron is closed. This will also reduce the number of changes in #1444

Reply:
Ack yes, this is better. Thanks.